module/Olcs/src/Form/Model/Fieldset/CloseConversationText.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Olcs\Form\Model\Fieldset;
+
+use Laminas\Form\Annotation as Form;
+
+/**
+ * @Form\Name("main")
+ */
+class CloseConversationText
+{
+    /**
+     * @Form\Type("\Common\Form\Elements\Types\PlainText")
+     * @Form\Attributes({"value":"The conversation will be removed from the Inbox and a transcript will be archived in docs and attachments tab."})
```

Comment:
This is probably fine as-is, but wondering if it's worth making this text editable? Not even sure how we do that with form field labels

Reply:
As discussed, will look to do this later.